internal/test/wmcb/main_test.go

```diff
@@ -19,14 +19,14 @@ var (
 )
 
 func TestMain(m *testing.M) {
-	var vmCreds e2ef.Creds
+	var vmCreds credentials.Credentials
 	var skipVMSetup bool
 
 	flag.Var(&vmCreds, "vmCreds", "List of VM credentials")
 	flag.BoolVar(&skipVMSetup, "skipVMSetup", false, "Option to disable setup in the VMs")
 	flag.Parse()
 
-	err := framework.Setup(vmCount, vmCreds, skipVMSetup)
```

Comment:
Why are you removing `skipVMSetup`?

Reply:
added skipVMSetup back. removed vmCreds as we can no more take vm creds. Instead using the existing private key to access the VMs created.

Reply:
So given `vmCreds` cannot be used what is the use of  `skipVMSetup`? Or is there a way for us to use an existing `VM` / `Machine` that is not immediately obvious to me?

And please note, I asked `Why are you removing skipVMSetup`. I did not request you to add it back. Please think things through before making changes to a question.